android/app-newm/src/debug/java/io/newm/DebugNewmApplication.kt

```diff
@@ -0,0 +1,29 @@
+package io.newm
+
+import co.touchlab.kermit.Logger
+import com.facebook.flipper.android.AndroidFlipperClient
+import com.facebook.flipper.android.utils.FlipperUtils
+import com.facebook.flipper.plugins.crashreporter.CrashReporterPlugin
+import com.facebook.flipper.plugins.databases.DatabasesFlipperPlugin
+import com.facebook.flipper.plugins.inspector.DescriptorMapping
+import com.facebook.flipper.plugins.inspector.InspectorFlipperPlugin
+import com.facebook.flipper.plugins.navigation.NavigationFlipperPlugin
+import com.facebook.flipper.plugins.sharedpreferences.SharedPreferencesFlipperPlugin
+import com.facebook.soloader.SoLoader
+
+class DebugNewmApplication : NewmApplication() {
+    override fun onCreate() {
+        Logger.d { "NewmAndroid - Debug Newm Application" }
+
+        SoLoader.init(this, false)
+        val client = AndroidFlipperClient.getInstance(this)
```

Comment:
nit: maybe you could use `.apply { }` here to simplify all those `addPlugin` calls